packages/test/src/util/browser-tests-runner/util/wdio-defaults.js

```diff
@@ -0,0 +1,53 @@
+const { env } = process;
+const path = require("path");
+
+module.exports = options => {
+  const wdioOptions = (options.wdioOptions = options.wdioOptions || {});
+
+  // When a custom launcher is used skip default values.
+  if (wdioOptions.launcher) {
+    return;
+  }
+
+  let { launcher, ...defaults } = env.BROWSERSTACK_USERNAME
```

Comment:
I'm not typically picky about syntax, but I really am not a fan of nested ternary operations. It's definitely not more readable in my opinion. Thoughts on changing this to use if statements?

Reply:
I’ll think about it.